fix(woocommerce): fix patch failure on some sites

karthikeyan5 opened the pull request:
MYSQL error: 1093 was occurring on some sites when running this patch.

rmehta commented on erpnext/patches/v11_1/woocommerce_set_creation_user.py:
```diff
@@ -4,7 +4,13 @@
-					WHERE doctype = 'Woocommerce Settings'
-					AND field = 'creation_user';""")
\ No newline at end of file
+		modified_by = frappe.db.sql("""SELECT t.value
```
please no hanging indents

https://github.com/frappe/erpnext/wiki/Coding-Standards#use-simple-indenting

karthikeyan5 replied:
it does look strange on github... below is how it looked on vscode:
![hanging_SQL](https://user-images.githubusercontent.com/4847317/56292119-2fda0b00-6144-11e9-81fe-f9687c423892.png)

I have updated it now...

karthikeyan5 committed:
fix(woocommerce): fix patch failure on some sites

> ORM cannot be used as an error occurs only on Travis.
https://travis-ci.com/frappe/erpnext/jobs/193285704

karthikeyan5 commented:
already fixed by https://github.com/frappe/erpnext/pull/17250
